packages/legend-studio/src/stores/GraphState.ts

```diff
@@ -186,6 +186,14 @@ export class GraphState {
     );
   }
 
+  createEmptyGraph(): PureModel {
```

Comment:
Hm, where do we use this?

Reply:
Anyway, it should not be here I think, let's move it where we need it

Reply:
we had discussed this. Graph State makes the most sense. You are creating an empty graph but with the core and system models from graph state

Reply:
I just realized we need this function at other places in graph state as well so this makes sense. Will leave it in `GraphState`. Just my knee-jerk reaction when people add more things to `GraphState`